New Tutorials on How to Cite Galaxy and Integrate InvenioRDM Repositories with Galaxy (#6420)
galaxyproject/training-material: topics/introduction/tutorials/galaxy-cite/tutorial.md

---
layout: tutorial_hands_on
title: Best Practices for Citing Galaxy
tags:
  - fair

>   - introduction

Reviewer: Not sure if the introduction fits as a tag in the context of the GTN.

level: Introductory
questions:
  - How to cite Galaxy in Publications?
objectives:

>   - Understand how to cite Galaxy.

Reviewer: Suggestion to have a more measurable action for the Objectives

  - Prepare your methodology section based on your Galaxy Histories.

> time_estimation: 10m

Reviewer: I would increase the time, maybe to 15 or 20 minutes. I know there are not really many hands-on parts, but there are some links that could also take time to go over.

> key_points:

Reviewer: I would formulate it more like key learnings of the tutorial and less like recommendations. AI suggestion:
- Cite the Galaxy Project: Always reference the primary Galaxy publication.
- Acknowledge the platform: Include the specific Galaxy server you used for your analyses.
- Share your work: Publish your Galaxy Histories and Workflows to make your methods transparent and reproducible.
- Credit your data sources: Cite any public datasets used in your analyses.
- Include tool citations: Export a list of tool citations directly from your Galaxy History.

Not sure if I would use it like this. But maybe a starting point for updating them. :)

  - Reference the specific Galaxy server that was used.
  - When preparing your methodology, we recommend publishing your Galaxy Histories and Workflows.
  - Remember to cite any public data used in your analysis.
  - You can extract a list of tool citations from your Galaxy History.
subtopic: core
contributions:
  authorship:
    - tflowers15
    - GarethPrice-Aus
    - mirelaminkova
    - MarisaJL
  funding:
    - unimelb
    - melbournebioinformatics
    - AustralianBioCommons
    - surf

---

# Overview

This is a short primer on how to cite Galaxy in your publication and also how to prepare your Methodology based on your Galaxy history or histories. Following these steps will make it easy to cite Galaxy, share your methods, and adhere to best practices for FAIR and open science.

> <agenda-title></agenda-title>
> In this tutorial we will deal with:
>
> 1. TOC
> {:toc}
>
{: .agenda}

# Citing Galaxy
Citing Galaxy comes in two main actions: 

1. Citing the global Galaxy Project that provides the core code that all Galaxy instances operate on. 
2. Citing the instance of Galaxy you did your work on.

## Citing Galaxy Project

> Please cite Galaxy Project’s primary publication: 

Author: I could see users interpreting your wording as citing the Galaxy Project publication as being optional and that they can choose not to. While we cannot force users to cite the Galaxy Project publication, I think this line should be clear that users should be citing the publication by keeping the wording as just the request that they citing the primary publication.

This is also found on the Galaxy Project Hub with more specific examples of how to cite individual aspects of Galaxy Project - [https://galaxyproject.org/citing-galaxy/](https://galaxyproject.org/citing-galaxy/)

> <comment-title></comment-title>
> The main Galaxy Project publication will be included in the list if you follow the instructions below to extract tool citations from your History.
>
{: .comment}

## Citing a Galaxy Service

### Service Descriptions
Each public Galaxy service benefits from being able to track the amazing work done by researchers using its service. Provided below are the DOIs for the Galaxy Australia, France and Main `usegalaxy.*` services. The linked papers are available on Zenodo in the Galaxy Project Zenodo Community at [https://zenodo.org/communities/galaxy/](https://zenodo.org/communities/galaxy/):

- Galaxy Australia - [https://doi.org/10.5281/zenodo.17298823](https://doi.org/10.5281/zenodo.17298823)
- Galaxy France - [https://doi.org/10.5281/zenodo.17298858](https://doi.org/10.5281/zenodo.17298858)

> - Galaxy Main - [https://doi.org/10.5281/zenodo.17298864](https://doi.org/10.5281/zenodo.17298864)

Author: Updated and new version of Galaxy Main service description published on Zenodo

We recommend including the relevant service name and DOI link in your Methods.

### Acknowledging a Service
Each public Galaxy service is provided through a complex mixture of funding, governance and infrastructure providers. As such, each service provides a cut/paste **Acknowledgement** statement that supports their particular configuration and this can be found on each service. If your publication supports Acknowledgements, we strongly recommend you include the service-recommended **Acknowledgements**.

**Acknowledgement** statements for the four main `usegalaxy.*` services can be found using the following links:

- Galaxy Australia - [https://site.usegalaxy.org.au/about#acknowledgement-statement](https://site.usegalaxy.org.au/about#acknowledgement-statement)
- Galaxy Europe - [https://usegalaxy-eu.github.io/about](https://usegalaxy-eu.github.io/about)
- Galaxy France - [https://www.france-bioinformatique.fr/en/services/acknowledgement/](https://www.france-bioinformatique.fr/en/services/acknowledgement/)

> - Galaxy Main - [https://usegalaxy.org/#about](https://usegalaxy.org/#about)

Reviewer: I do not see a description of what should be added to the Acknowledgment. Is there maybe a better link? Or did I miss something?

## Making a History public

We provide the following recommendations and process for making your History persistent and your options for public archiving of your History. Sharing your History will make it easier for others to understand or replicate your analysis. You can also add **annotations** and **tags** to ensure your history aligns to the FAIR principles.

### Preparing a History

We recommend cleaning up your History when preparing to make your History public.

> <hands-on-title>Clean up Unneeded Datasets</hands-on-title>
>
> - `Delete` and `Delete (permanently)` (purge) any failed jobs.
> - `Delete` and `Delete (permanently)` (purge) any datasets/job outputs that were used for testing and are not required for the final, published analysis.
> - `Delete` and `Delete (permanently)` (purge) any datasets that you don't want to share publicly (e.g. restricted data, unpublished results) - but make sure you save them to a private history first.
> 
> {% snippet faqs/galaxy/datasets_deleting.md %}
>
{: .hands_on}

We also suggest providing a description/context of the History in the `History Annotation`.

> <hands-on-title>Annotating a History</hands-on-title>

> >

Reviewer: I know they should follow the instructions in the tips. But I think it is nicer if we repeat the task they should do in the hands on box.

> <hands-on-title>Adding Tags to a History</hands-on-title>
>
> {% snippet faqs/galaxy/histories_tagging.md %}
>
{: .hands_on}

### Sharing a History

Galaxy provides a number of options for sharing histories which allows others to import and access the datasets, parameters, and steps of your history.

> <hands-on-title>Sharing a History</hands-on-title>
>
> {% snippet faqs/galaxy/histories_sharing.md %}
>
{: .hands_on}

### Exporting a History to Zenodo

Integration of InvenioRDM and Zenodo into Galaxy now enables you to export your history directly to Zenodo. Publishing your history on Zenodo will generate a DOI for your history which you can include in your publication and can be used by other researchers to cite your work.

For instructions on connecting to Zenodo and preparing your history, see the following tutorial on the Galaxy Training Network:

- [Integrating InvenioRDM-compatible Repositories with Galaxy]({% link topics/fair/tutorials/inveniordm-integration/tutorial.md %})

> ## Making a Workflow public

Author: Good point. I have updated the text and added a new line and hands-on box with the instructions for how to extract a workflow from a history.

If you used a workflow in your analysis, we recommend publishing this alongside your data and in your article. In addition to providing a complete description of your method, the published workflow will also make your approach reproducible and citeable.

Two repository options that we recommend for publishing your Workflows are:

- WorkflowHub ([https://workflowhub.eu/](https://workflowhub.eu/))
- Dockstore ([https://dockstore.org/](https://dockstore.org/)) 

For instructions on preparing your workflow, see the following tutorial on the Galaxy Training Network:

- [Annotate, prepare tests and publish Galaxy workflows in workflow registries]({% link topics/galaxy-interface/tutorials/workflow-fairification/tutorial.md %})

## Citing Public Data

If you used a public data source in your work, we recommend you cite both the data source publication DOI and the dataset itself. Most citation styles will include options for citing datasets. You should include the repository name and accession code in your Methods section to make it easy to find.

## Extracting Tool Citations from a Galaxy History

> Galaxy can provide a list of the tools used in your history using the following steps. You might find this helpful when writing up your methods, but it is also the quickest way to add all these tools to your references.

Reviewer: ```suggestion
Galaxy can all citations of the tools used in your history. You can directly extract them from your history. It is the quickest way to add all these tools to your references, and you might find this helpful when writing up your methods.
```

> <hands-on-title>Extracting Tool Citations</hands-on-title>

> >

Reviewer: ```suggestion
>
> 1. Export a list of tool citations from your History
>
```
